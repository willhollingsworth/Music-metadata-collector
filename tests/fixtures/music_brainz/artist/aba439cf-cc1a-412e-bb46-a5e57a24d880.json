{
  "isnis": [],
  "type": "Person",
  "disambiguation": "Electronic Music Producer \"Theo Vicente\" from Montpellier, France.",
  "gender-id": "36d3d30a-839d-3eda-8cb3-29be4384e4a9",
  "id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
  "gender": "Male",
  "type-id": "b6e035f4-3ce9-331c-97df-83397230b0df",
  "end-area": null,
  "name": "Teho",
  "life-span": {
    "end": null,
    "ended": false,
    "begin": null
  },
  "area": {
    "disambiguation": "",
    "name": "Montpellier",
    "type": null,
    "type-id": null,
    "id": "c7ee402c-0137-4931-a72e-a262796c0d16",
    "sort-name": "Montpellier"
  },
  "sort-name": "Teho",
  "country": "FR",
  "begin-area": null,
  "ipis": []
}