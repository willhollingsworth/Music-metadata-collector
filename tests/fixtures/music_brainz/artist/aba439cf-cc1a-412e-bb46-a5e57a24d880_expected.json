{
  "artist_name": "Teho",
  "artist_id": "aba439cf-cc1a-412e-bb46-a5e57a24d880",
  "type": "Person"
}